runtime/executor/src/main/java/edu/snu/onyx/runtime/executor/data/filter/Chainable.java

```diff
@@ -0,0 +1,29 @@
+package edu.snu.onyx.runtime.executor.data.filter;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+/**
+ * A {@link Chainable} object indicates each stream manipulation strategy.
+ * Stream can be chained by {@link Chainable} multiple times.
+ */
+public interface Chainable {
```

Comment:
How about `StreamChainer`?
If the name of this interface is `Chainable`, it should represent all general "chainable" classes.